addons/website/static/src/js/editor/widget_link.js

```diff
@@ -26,43 +28,88 @@ weWidgets.LinkTools.include({
     },
     /**
      * Allows the URL input to propose existing website pages.
-     *
+     * Allows the URL input to be filled from document upload.
+     * Automatically creates a link if selected content is a phone number.
+     * 
      * @override
      */
     start: async function () {
         var def = await this._super.apply(this, arguments);
-        wUtils.autocompleteWithPages(this, this.$('input[name="url"]'));
+        this.$urlInput = this.$('#o_link_dialog_url_input');
+        wUtils.autocompleteWithPages(this, this.$urlInput);
         this._adaptPageAnchor();
+        /*
+         * Strips spaces and parenthesis from link text.
+         * If text is a phone number (only composed of digit except for leading +), make the link callable.
```

Comment:
This is not working in `start` method, `_onURLInput` seems more appropriate or something that fired on link save?
Steps I did:
- Select Text
- Create link (Ctrl-K) -> Start() fire but no link yet since I didnt type yet
- TYpe phone
-> The code is not called again, the created link is `http://04554343333`

Reply:
It is not supposed to work this way, at least the way I did it (which can obv be wrong). You should type a phone number as text, select it and create a link from this.

https://www.awesomescreenshot.com/video/3494226 

I showed this flow to @sbuhl  whom didn't seem to disagree with it, I'll ask again.

Reply:
I cannot access the link
Well, I don't think it makes sense to not handle the flow I tried IMHO.
Imagine you had a typo in your phone, you change it, it won't work since you would need to edit it 2 times.
Plus lot of cases where you create a link without typing the phone as plain text, like a link button, a link icon (a fa-phone), a plain text like "Click here to call us" etc
What do you think

Reply:
Indeed, this makes sense. I should handle both flows then ! I'm on it ;)